config/manifests/bases/dynatrace-monitoring.clusterserviceversion.yaml

```diff
@@ -0,0 +1,326 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - monitoring
+  - dynatrace
+  - oneagent
+  links:
+  - name: Operator Deploy Guide
+    url: https://www.dynatrace.com/support/help/shortlink/openshift-deploy
+  - name: OpenShift Monitoring Info
+    url: https://www.dynatrace.com/technologies/openshift-monitoring
```

Comment:
Kubernetes and OpenShift have different links.

Reply:
In the base file I can't provide two of them